istio/setup.cfg

```diff
@@ -0,0 +1,9 @@
+[aliases]
+test = pytest
+
+[tool:pytest]
+testpaths = ./tests
+
+[flake8]
```

Comment:
with @ofek we agreed to to kill the `setup.cfg` file and move the `[flake8]` section to `tox.ini`

Reply:
@masci @ofek  is there an example of the setup you wanna use?

Reply:
nm figured it out!